src/store/creator/reducer.ts

```diff
@@ -1,12 +1,13 @@
 import { createReducer } from '@reduxjs/toolkit';
 import { CreatorState, CreatorStatus } from './types';
 import { createAnnotationAction } from '../annotations';
-import { setMessageAction, setCursorAction, setStagedAction, setStatusAction } from './actions';
+import { setMessageAction, setCursorAction, setSelectionAction, setStagedAction, setStatusAction } from './actions';
 
 export const initialState = {
     cursor: 0,
     error: null,
     message: '',
+    selection: null,
```

Comment:
Is this an appropriate place for this property? It is only used by a single annotator currently and its selector is not yet used. It also seems to somewhat duplicate the `staged` property.